frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/clearingStatus.jspf

```diff
@@ -75,7 +75,7 @@
                 {title: "Project Origin"},
                 {title: "Clearing State"},
                 {title: "Mainline State"},
-                {title: "Project Mainline State"},
+                {title: "Project Mainline state"},
                 {title: "Action", data: "DT_RowId", render: renderActions}
```

Comment:
I think it was a mistake by me: please let"s start all letters with a capital letter